.github/CHANGELOG.md

```diff
@@ -50,14 +50,16 @@
 1. [FMGC] Fix unnatural turn direction for path capture transition - @tracernz (Mike)
 1. [FMGC] Fix unnatural turn direction for course capture transition - @tracernz (Mike)
 1. [FMGC] Update spoiler CD for MSFS SDK clarification - @donstim (donbikes#4084)
-1. [HYD] Actuators damping chambers can be asymetrical - @Crocket63 (crocket)
```

Comment:
Careful, you dropped somebody else's changelog.

Reply:
I think it's due to the fact the other PRs are not merged yet